_projects/roberta_external_source.md

```diff
@@ -14,7 +15,8 @@ mentors:
 name: "One click linting for online projects in the browser"
 requirements:
   - "The applicant should be be familiar with bear creation and XML parsing using xpath or a similar query language"
-  - "The applicant should be reasonable familiar with the coala codebase and the workings of REST APIs, including authentication."
+  - "The applicant should be reasonable familiar with the coala codebase and the workings of REST APIs, including
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.